### ISSUE TITLE
Code trying to push to fields bug

### ISSUE THREAD

**Issue by driskull** [Esri/geoform-template-js#420]
@CTDisasterManagement can you verify this fix is correct?

**Comment by CTDisasterManagement** [comment]
@driskull,

The code has been moved to "startup" function in main.js.If application url contains "edit" parameter then this code will not execute and hence rtl css will not be included in author mode.

Previously the code was present in constructor, so even if application url has "edit" parameter. main.js was parsed and through its constructor "rtl.css" was getting applied. 
We just reverted the code and #354 is working fine.

Thanks!

**Comment by driskull** [comment]
Ok thanks!

- Matt


On Feb 17, 2015, at 10:43 PM, CTDisasterManagement <notifications@github.com<mailto:notifications@github.com>> wrote:


@driskull<https://github.com/driskull>,

The code has been moved to "startup" function in main.js.If application url contains "edit" parameter then this code will not execute and hence rtl css will not be included in author mode.

Previously the code was present in constructor, so even if application url has "edit" parameter. main.js was parsed and through its constructor "rtl.css" was getting applied.
We just reverted the code and #354<https://github.com/Esri/geoform-template-js/issues/354> is working fine.

Thanks!
